packages/react-server/core/logging/client.js

```diff
@@ -10,13 +10,35 @@
 
 var common = require('./common')
 
-// IE9 doesn't even _define_ console unless the dev tools are open.
 var _console = (typeof console === 'undefined')
-	?{log:()=>{}}
-	:console;
-
-// This is just so we have a real function to work with in IE9.
-var console_log = Function.prototype.bind.call(_console.log, _console);
+	?{
+		// IE9 doesn't even _define_ console unless the dev tools are open.
+		log   : () => {},
+		error : () => {},
+		warn  : () => {},
+		debug : () => {},
+		info  : () => {},
+	}
+	:{
+		// IE9 also needs a real function for `apply` to work.
+		log   : Function.prototype.bind.call(console.log,   console),
+		error : Function.prototype.bind.call(console.error, console),
+		warn  : Function.prototype.bind.call(console.warn,  console),
+		debug : Function.prototype.bind.call(console.debug, console),
+		info  : Function.prototype.bind.call(console.info,  console),
+	};
+
+var lvl_map = {
+	emergency : 'error',
+	alert     : 'error',
+	critical  : 'error',
+	error     : 'error',
+	warning   : 'warn',
```

Comment:
Yep.  This only addresses routing of existing `logger` methods.  It doesn't add any new ones.